### PR TITLE
add support for tiev2

### PR DESCRIPTION
**Pull request by hannahw0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,16 +42,18 @@ Important: when after saving an error appears in the Splunk Web tool, the config
...
+can be retrieved on *https://developer.dcso.de/*. 
+If you have any questions about the credentials, please contact DCSO (see below).
 
 There are few more details about the configuration:
 
-* **API Token**: either a legacy tie.dcso.de token, or new one created through the DCSO Portal.
-* **Initial IoC Sequence Number**: Use this to start from a particular sequence number. This is
-  useful when re-installing or upgrading to incompatible add-on version (data in index would
-  stay compatible). Leave 0 to use whatever number is stored or start from NOW minus 30 days. 
+* **API Client ID**: DCSO api client_id from the DCSO Developer Portal.
```
Comment (ctc-phil) [inline README.md:51]: is developer portal the only way to get credentials? i would not assume so. maybe reference idm credentials instead

File: lib/dcsotiesplunk/config.py [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
+        "url_severity": (SeverityRange, 1),
+        "email_confidence": (ConfidenceRange, 80),
+        "email_severity": (SeverityRange, 1),
+        "confidence": (ConfidenceRange, 80),
+        "severity": (SeverityRange, 2),
+    },
+    "tie": {
+        "client_id": (str, ""),
+        "client_secret": (str, ""),
+        "feed_api": (str, "https://api.dcso.de/tie/v1/iocs"),
```
Comment (ctc-phil) [inline lib/dcsotiesplunk/config.py:24]: is this feed api string still the same?

Comment (hannahw0) [reply lib/dcsotiesplunk/config.py:24]: not sure what you mean... this is the new api string

File: README.md (hunk 1 above)
Comment (hannahw0) [reply README.md:51]: good point

File: lib/dcsotiesplunk/config.py (hunk 2 above)
Comment (fhonza) [reply lib/dcsotiesplunk/config.py:24]: perhaps: tie/v1/iocs -> is still the Tie-v1 used?

Comment (satta) [reply lib/dcsotiesplunk/config.py:24]: This uses the new TIE software (note that it's `api.dcso.de`, not `tie.dcso.de`) but its  "v1" API flavour, not the TAXII ("v2") one. TAXII is currently not in the state to be used productively for a case like this.
The URL is correct.

**Comment by ctc-phil** [comment]
@satta merge is blocked because hanna did not commit via ssh

we can merge anyways, right?

**Comment by satta** [comment]
Well I didn't test it, but whatever ssh requirement there might be should not be a blocker.

**Pull request merged**